enterprise/utils.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities to be used across the enterprise djangoapp
```

Comment:
All right - certainly doable. I was looking for existing patterns to follow, and a utils module seems typical.